index.html

```diff
@@ -3,18 +3,68 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | Haseeb Khalil (Haseeb3927)</title>
+    <!--Font style-->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <!--CSS-->
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
 
-  <body>
-    <div id="root">
+  <body id="bodyEl">
+    <!--Page heading H1-->
+    <h1 id="mainHeading">TV Show</h1>
+
+    <!--Search header-->
+    <div id="header">
+      <select id="showSelection">
+        <!--All shows will go here-->
+      </select>
+
+      <select id="episodeSelection">
+        <option value="selected">
+          <selected>Select a show to see episodes</selected>
+        </option>
+        <!--All episodes will go here-->
+      </select>
+
+      <!--Search Bar-->
+      <input
+        type="text"
```

Comment:
You can use input of type="search", this really has no differences to text but it comes with a handy x button to clear the search bar. Personal preference but I like it so I wanted to mention it!

Reply:
Thank you